docs/source/tools/acquire.rst

```diff
@@ -63,6 +72,44 @@ These arguments do the following:
 
 You can specify these arguments multiple times for every file, directory or glob you want to collect.
 
+Volatile acquisition
+--------------------
+
+*new in Acquire 3.11*
+
+Use ``--volatile-profile`` to obtain artefacts that are not persistent on disk but are located in RAM.
+Volatile Windows artifacts are stored in de ``$metadata$`` folder in the resulting archive.
+Windows volatile artifacts are acquired through the use of internal Windows commands and the Python ``ctypes`` interface.
+For Linux systems, ``proc`` and ``sys`` are acquired and stored under ``/proc/1/...`` or ``/sys/fs/...`` respectively.
+
+Volatile Profiles
+^^^^^^^^^^^^^^^^^
+
+Like regular *profiles*, the *volatile profiles* allow you to run predefined groups of modules.
+The following volatile profiles are available:
+
+* ``default``: Collect a set of volatile information.
+* ``extensive``: Collect a lot, if not all, the volatile information. Usually the modules that either take a lot of time or are very involved.
+* ``none``: Do not collect any volatile information.
```

Comment:
I think it is a good idea to make the explanation the same format as done with acquire profiles. E.g. listing the available, directly followed by the contents of the profile, so the user immediately sees which profile does what. See line 21 and up in `acquire.rst` for the example.

That way, you avoid having to talk about the profile like this. Which currently, raises more questions for me reading it like this :)